employee-rostering-frontend/src/store/tenant/tenant.test.ts

```diff
@@ -41,31 +42,69 @@ describe('Tenant operations', () => {
       name: "Tenant 3"
     }];
 
+    let expectedRefreshActions: any[] = [
```

Comment:
My advice: avoid using `any` unless necessary. It's something like `Object` in Java. Actual type of this variable is `(RefreshTenantListAction | RefreshSkillListAction | RefreshSpotListAction)[]` but that can be inferred from the value, so you don't have to specify it explicitly. In most simple cases it's really unnecessary to specify the variable type. For example in `const name: string = "Jiri";` you can safely omit `: string` type annotation because it's obvious that the type of the variable is `string`.

Reply:
I reuse `expectedRefreshActions` later where the first element is an `ChangeTenantAction`, hence the inferred type becomes an issue (and why I use `any`). So I would need to specify the type as `(ChangeTenantAction | RefreshTenantListAction | RefreshSkillListAction | RefreshSpotListAction)[]`.

Reply:
True. I missed the `ChangedTenantAction`. In that case I'd say it's better not to reuse the variable a declare a dedicated one for each set of expected actions.